charts/intel-gpu-resource-driver/templates/clusterrole.yaml

```diff
@@ -1,8 +1,8 @@
-{{- if and .Values.rbac.create }}
+{{- if .Values.rbac.create }}
```

Comment:
Same question as in https://github.com/intel/helm-charts/pull/57#discussion_r1691396061 : does the resource driver work without the RBAC? What will happen when user inevitably selects not to install RBAC rules? Why are we giving this option ?